src/components/Layout/Header/ActionCenter/components/GenericTransactionActionCard.tsx

```diff
@@ -52,8 +54,14 @@ export const GenericTransactionActionCard = ({ action }: GenericTransactionActio
       defaultExplorerBaseUrl: feeAsset.explorerTxLink,
       address: undefined,
       maybeSafeTx: undefined,
+      txStatus: action.status === ActionStatus.Pending ? TxStatus.Unknown : TxStatus.Confirmed,
```

Comment:
ditto q here

Reply:
I removed this in [19028ad](https://github.com/shapeshift/web/pull/10171/commits/19028ad2d0d86ce479a345910236f9545189631c)

Focusing on swaps only